tools/deb/envoy_bootstrap_tmpl.json

```diff
@@ -0,0 +1,121 @@
+{
+  "stats_config": {
+    "use_all_default_tags": false
+  },
+  "admin": {
+    "access_log_path": "/dev/stdout",
+    "address": {
+      "socket_address": {
+        "address": "127.0.0.1",
+        "port_value": 15000
+      }
+    }
+  },
+  "dynamic_resources": {
+    "lds_config": {
+      "api_config_source": {
+        "api_type": "REST_LEGACY",
+        "refresh_delay": {{ .refresh_delay }},
+        "cluster_names": [
+          "rds"
+        ]
+      }
+    },
+    "cds_config": {
+      "api_config_source": {
+        "api_type": "REST_LEGACY",
+        "refresh_delay": {{ .refresh_delay }},
+        "cluster_names": [
+          "rds"
+        ]
+      }
+    },
+    "deprecated_v1": {
+      "sds_config": {
+        "api_config_source": {
+          "api_type": "REST_LEGACY",
+          "refresh_delay": {{ .refresh_delay }},
+          "cluster_names": [
+            "rds"
+          ]
+        }
+      }
+    }
+  },
+  "static_resources": {
+    "clusters": [
+      {
+        "name": "rds",
+        "type": "STRICT_DNS",
+        "connect_timeout": {{ .connect_timeout }},
+        "lb_policy": "ROUND_ROBIN",
+{{ if eq .config.ControlPlaneAuthPolicy 1 }}
+      "tls_context": {
+        "common_tls_context": {
+          "tls_certificates": {
+            "certificate_chain": {
+              "filename": "/etc/certs/cert-chain.pem"
+            },
+            "private_key": {
+              "filename": "/etc/certs/key.pem"
+            }
+          },
+          "validation_context": {
+            "trusted_ca": {
+              "filename": "/etc/certs/root-cert.pem"
+            },
+            "verify_subject_alt_name": [
+              "spiffe://cluster.local/ns/istio-system/sa/istio-pilot-service-account"
```

Comment:
these are customizable I believe, cc @myidpt

Reply:
They seem hardcoded in current pilot.